src/UI/Core/TextField.cpp

```diff
@@ -139,13 +139,13 @@ void TextField::onTextInput(const std::string& _s)
 
 	if (mMaxCharacters > 0 && text().length() == mMaxCharacters) { return; }
 
-	int prvLen = text().length();
+	std::size_t prvLen = text().length();
```

Comment:
No issue with this change. I actually prefer this sort of solution. Just want to point out that many other places currently using `static_cast` may be due to an improper type being used. We should at some point look at the uses of `static_cast` and see if a change in type would perhaps work better.

Reply:
Just noticed this change is also included in #174. It maybe fits better in the other PR.

Reply:
Done